Subject: also unload OSLEC (echo) on module unload
From: Tzafrir Cohen <tzafrir.cohen@xorcom.com>
Forwarded: no

This patch sets the DAHDI init.d script to also unload 'echo' (OSLEC)
in the 'stop' target. This adds to the default of unloading dahdi and
all of its dependent modules.

The package dahdi-linux includes 'echo' and with the default settings,
will attempts load 'echo', that does not depend on dahdi, and hence
will not be unloaded by default. Thus upgrading to a newer version (of
a newer build) may break.

This change will not be commited upstream, as OSLEC is not used there.

--- a/dahdi.init
+++ b/dahdi.init
@@ -39,7 +39,7 @@ DAHDI_DEV_TIMEOUT=20
 
 # A list of modules to unload when stopping.
 # All of their dependencies will be unloaded as well.
-DAHDI_UNLOAD_MODULES="dahdi"
+DAHDI_UNLOAD_MODULES="dahdi echo"
 
 #
 # Determine which kind of configuration we're using
